svelte-kit-scss/src/lib/components/auth/Auth.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+import Auth from './Auth.svelte';
+
+describe('Auth Component', () => {
+  it('should render sign in', async () => {
+    render(Auth);
+
+    const signInBtn = screen.getByRole('button');
+    expect(signInBtn.textContent).toEqual('Sign in with GitHub');
+    await userEvent.click(signInBtn);
+
+    expect(document.location.href).toEqual('http://localhost:3000/');
```

Comment:
This test would not work in a CI environment or production/staging environments if you ever needed tests there for whatever reason since the path would potentially not be `localhost`. Instead you could check for the relative path that is returned.

## Suggestion
Use `window.pathname` to check if it returns us to our root application path (`/`).